### ISSUE TITLE
Item thumbnails exist but do not load on initial route

### ISSUE THREAD

**Issue by AndrewZWood** [DSpace/dspace-angular#1046]
When I generate thumbnails for items they do not load initially. I can refresh the page but even after I've cleared all related cache I have to re-route the the item for the thumbnail to display. 

To reproduce:
Submit a new item.
Run the filter media process.
Route to your new item.
Verify the "No Thumbnail" default display.
Press the back button.
Re-route to the your new item. 
Notice the thumbnail now displays.

It's unclear to me why re-routing to an item after initially routing to the item would display the thumbnail.

**Comment by cwilper** [comment]
Note: At first glance this might look similar to https://github.com/DSpace/dspace-angular/issues/797 but it is different: This is a case where thumbnails that were generated long ago (many restarts and browser cache clears) will not display on initial page display, and only after you browse through the interface are they finally displayed for the user.

**Comment by tdonohue** [comment]
@cwilper or @AndrewZWood : Have you done any further analysis of the issue here?  This seems like something we may want to solve in 7.0, but also seems low priority.  If you don't feel experienced enough to figure out the issue, we could also pull in @artlowel or someone on his team.

For now, moving this to 7.0 board, flagging as low priority & assigning back to @cwilper and @AndrewZWood (please give feedback on whether I should reassign it though)

**Comment by tdonohue** [comment]
Per today's meeting, I'm reassigning this to @artlowel so that he can delegate to appropriate team members to analyze the issue.

**Comment by MarisArins** [comment]
Hi, I would like to add an extra scenario that I see and it could be very much related to this bug.
1. I navigate to an item with thumbnail via links
2. Thumbnail is displayed in place as supposed.
3. I hit page refresh (regular and full refresh without cache usage all the same) 
4. Thumbnail does not get loaded (No thumbnail svg in place)

This also has same effect if I pass the item URL to someone else - every attempt to load item fully fails.

P.S.
did not found any thumbnails on demo site, so this is only observed on local installation
